scripts/ironic/playbooks/ironic-post-install-config.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- name: Ironic post install config
+  hosts: utility_all[0]
+  tasks:
+    - name: Setup neutron network
+      shell: ". /root/openrc && neutron net-create --shared --provider:physical_network tftp --provider:network_type flat tftp"
+
+    - name: Setup neutron subnet
+      shell: ". /root/openrc && neutron subnet-create --name ironic-tftp --allocation-pool start=172.29.233.0,end=172.29.234.0  --dns-nameserver=4.4.4.4 tftp 172.29.232.0/22"
+
+    - name: Configure the cleaning network
+      delegate_to: localhost
+      config_template:
+         src: "/etc/openstack_deploy/user_osa_variables_overrides.yml"
+         dest: "/etc/openstack_deploy/user_osa_variables_overrides.yml"
+         owner: "root"
+         group: "root"
+         mode: "0644"
+         config_overrides: "{{ ironic_cleaning_config }}"
+         config_type: "yaml"
+  vars:
+    ironic_cleaning_config:
+      ironic_neutron_cleaning_network_name: "tftp"
+      ironic_ironic_conf_overrides:
+        conductor:
+          automated_clean: true
```

Comment:
depending on what is done with the ironic nodes this may make gate slower (clean on initial add and return to pool).